<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportConfigWindow</class>
 <widget class="QDialog" name="ImportConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import file</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="aliasLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameTxt"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Import Source</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="importSourceCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Existing File</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VMess and QRCode</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Subscription Link</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Import</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="filePage">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="fileLabel">
            <property name="text">
             <string>Path</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="fileLineTxt">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="selectFileBtn">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Inbound</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="keepImportedInboundCheckBox">
            <property name="text">
             <string>Keep imported inbound settings</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Edit Config</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Open in JSON Editor</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="vmessPage">
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>6</x>
            <y>6</y>
            <width>153</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>VMess Connection String</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>227</width>
            <height>36</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="text">
              <string>---TODO---</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>504</x>
            <y>80</y>
            <width>40</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Image</string>
          </property>
         </widget>
         <widget class="QLabel" name="qRContentLabel">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>104</y>
            <width>16</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>C</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>434</x>
            <y>104</y>
            <width>110</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Connection String</string>
          </property>
         </widget>
         <widget class="QPushButton" name="qrFromScreenBtn">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>40</y>
            <width>84</width>
            <height>34</height>
           </rect>
          </property>
          <property name="text">
           <string>Go</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>437</x>
            <y>40</y>
            <width>107</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Scan from Screen</string>
          </property>
         </widget>
         <widget class="QLabel" name="imageLabel">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>80</y>
            <width>16</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>I</string>
          </property>
         </widget>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>80</y>
            <width>404</width>
            <height>194</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
           <item>
            <widget class="QTextEdit" name="vmessConnectionStringTxt">
             <property name="placeholderText">
              <string>Put your vmess:// here</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>VMess Check Status</string>
             </property>
             <layout class="QFormLayout" name="formLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Processing</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>0/0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Line Failed</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Subscription Link</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="subscriptionLinkTxt">
            <property name="text">
             <string>TO DO IN VERSION 2</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
            <item>
             <widget class="QTableWidget" name="tableWidget"/>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>GroupBox</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fileLineTxt</tabstop>
  <tabstop>selectFileBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportConfigWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportConfigWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
